src/bindings/python/flux/modprobe.py

```diff
@@ -16,7 +16,7 @@
 import sys
```

Comment:
Commit message:
s/buliting/built-in/
s/subpackage/framework package/

> It is backwards compatible with the old system of always dropping
   files into

Maybe "allowing drop-in files in..." was meant?